frontend/src/pages/TermPlanner/TermBox/TermBox.tsx

```diff
@@ -82,9 +83,29 @@ const TermBox = ({ name, coursesList, termsOffered, dragging }: Props) => {
               ref={provided.innerRef}
               {...provided.droppableProps}
             >
-              {coursesList.map((code, index) => (
-                <DraggableCourse key={`${code}${term}`} code={code} index={index} term={term} />
-              ))}
+              {coursesList.map((code, index) => {
+                if (multiCourseDrag !== '' && multiCourseDrag === code) {
```

Comment:
can remove this if statement and just pass in `isDragged={multiCourseDrag === code`